packages/config/config/devices/0x0330/VES-ZW-SOC-28.json

```
@@ -0,0 +1,78 @@
+{
+	"manufacturer": "Vesternet",
+	"manufacturerId": "0x0060",
+	"label": "VES-ZW-SOC-28",
+	"description": "UK Socket Plug",
+	"devices": [
+		{
+			"productType": "0x0034",
+			"productId": "0x0001",
+			"zwaveAllianceId": [4675]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Reports",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Control",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Remember State",
+			"description": "After power failure, resume the socket previous state",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled (socket stays off)",
+					"value": 0
+				},
+				{
+					"label": "Enabled (socket resumes previous state)",
+					"value": 1
+				}
+			]
```

We have a template for this
```suggestion
			"$import": "~/templates/master_template.json#state_after_power_failure_off_prev"
```